docs/docs/mlflow-3/index.mdx

```diff
@@ -0,0 +1,182 @@
+- Centrally track and analyze the performance of your models and generative AI applications **across all environments**, from interactive queries in a development notebook through production batch or real-time serving deployments.
+- Gain insights into model performance with **comprehensive metrics and lineage tracking**, enabling better decision-making for production readiness.
+- Streamline the handling of model artifacts and evaluation results with a **cohesive structure** that improves observability and traceability.
+
+Model Tracking helps you create and evaluate different model/agent configurations in your experiments, <APILink fn="mlflow.entities.LoggedModel">LoggedModel</APILink> helps you find the best ones for production usage, and tracing helps enhance observability in your model or GenAI applications.
+
+
+## Enhanced Model Tracking
+
+In MLflow 3.0, we introduce a refined architecture along with revamped APIs and UI, tailored to enhance generative AI and deep learning workflows. 
+With GenAI agents, there are multiple rounds of offline evaluation via batch jobs and interactive queries from human beta testers.
+In deep learning, training often generates multiple model checkpoints, where the best candidates are further evaluated before production deployment. 
+
+We are introducing a new first-class object, the <APILink fn="mlflow.entities.LoggedModel">LoggedModel</APILink> entity, into MLflow Tracking to streamline these processes. As you define and evaluate your GenAI agents in code, 
+or your deep learning jobs create and evaluate models, they will be automatically stored as MLflow Logged Models in your MLflow Experiment. 
+
+Each GenAI agent consolidates its evaluation metrics from both offline jobs and online interactions if you use managed MLflow with Databricks. 
+This feature removes the need to query Runs for traces from different environments (batch jobs, different notebook sessions etc.) to gain a full understanding of the agent’s performance.
+Similarly, every deep learning checkpoint is stored as a <APILink fn="mlflow.entities.LoggedModel">LoggedModel</APILink> with direct links to its own metrics. There is no longer any need to carefully log evaluation metrics for each checkpoint
+at its epoch all within the same Run, or to later query all metrics at a certain epoch just to retrieve metrics for one checkpoint. 
+
+Specifically, you will be able to:
+- View Logged Models alongside key information, including code and weights as artifacts and configurable parameters that affect model behavior
+- Holistically track all performance metrics related to your logged models with links to training and evaluation jobs
+- Search for and rank existing models based on a set of certain criteria, such as performance metrics on training or evaluation datasets
+
+### Quickstart
+
+This quickstart example demonstrates how to use the LoggedModel lineage feature in MLflow 3.0 with openai sdk, showcasing its integration with runs and traces.
```

Comment:
Can we explain that this example creates a generative AI application with prompt engineering and then evaluates it?